oauth2l/__init__.py

```diff
@@ -222,14 +225,18 @@ def _GetApplicationDefaultCredentials(scopes):
             return credentials.create_scoped(scopes)
 
 
+def _GetCredentialsFilename(credentials_filename):
+    return os.path.expanduser(credentials_filename or '~/.oauth2l.token')
+
+
 def _GetCredentialsVia3LO(client_info, credentials_filename=None):
-    credentials_filename = os.path.expanduser(
-        credentials_filename or '~/.oauth2l.token')
-    credential_store = multistore_file.get_credential_storage(
-        credentials_filename,
+    credentials_filename = _GetCredentialsFilename(credentials_filename)
+    storage_key = '3LO-{}-{}-{}'.format(
         client_info['client_id'],
         client_info['user_agent'],
```

Comment:
I thought there are some reason to put user agent so I didn't remove it. If it is not necessary I will remove it.